settlement.go

```diff
@@ -48,9 +48,13 @@ type TaxTotalAmount struct {
 
 // NewSettlement creates the ApplicableHeaderTradeSettlement part of a EN 16931 compliant invoice
 func NewSettlement(inv *bill.Invoice) *Settlement {
+	typeCode := TypeCodeInstrumentNotDefined
+	if !isValidTypeCode(typeCode) {
+		return nil
+	}
```

Comment:
I'm not sure about how we are handling typeCode here. It is being set to a concrete value and then being checked if the value is valid. I would suggest moving the decision on what typeCode to use to a function and return the TypeCodeInstrumentNotDefined until we implement other payment types. What do you think?

Reply:
Good idea. I wasn't sure how this could be handled.